guillotina/contrib/memcached/driver.py

```diff
@@ -160,3 +188,47 @@ async def flushall(self) -> None:
         for node in client.cluster_managment().nodes():
             with watch("flush"):
                 await client.flush_all(node)
+
+
+async def metrics_probe(client: emcache.Client, every: int = 10):
+    """
+    Periodically updates memcached cluster metrics
+    """
+    state: Optional[emcache.ConnectionPoolMetrics] = None
+    while True:
+        state = await update_connection_pool_metrics(client, state)
+        await asyncio.sleep(every)
+
+
+async def update_connection_pool_metrics(
+    client: emcache.Client, last_state: Optional[emcache.ConnectionPoolMetrics] = None
+) -> emcache.ConnectionPoolMetrics:
+    # Every node will have it's own label
+    metrics = client.cluster_managment().connection_pool_metrics()
+    for node, node_metrics in metrics.items():
+        MEMCACHED_CURRENT_CONNECTIONS.labels(node=node).set(node_metrics.cur_connections)
+
+        for counter, labels_to_attr in {
+            MEMCACHED_CONNECTION_POOL_CONNECTIONS: [
+                ("created", "connections_created"),
+                ("created_with_error", "connections_created_with_error"),
+                ("purged", "connections_purged"),
+                ("closed", "connections_closed"),
+            ],
```

Comment:
IMO I would prefer to have them as independent metrics, instead of using labels for identifying each of the counters. Makes the code more implicit IMO.